src/foam/nanos/dig/SUGAR.js

```diff
@@ -69,28 +70,17 @@ foam.CLASS({
 
           return;
         }
-        var of = this.lookup(service.cls_.getAxiomByName('delegate').of);
+        var of = foam.lookup(service.cls_.getAxiomByName('delegate').of);
 
         if ( ! of ) return;
 
         this.interfaceName = of.id.toString();
         var methods = of.getOwnAxiomsByClass(foam.core.Method);
-        // var methodName = methods.map(function(m) { return m.name; }).sort();
-
-        var filteredMethod =
-          methods.filter(function(fm) {
-            for ( var j = 0 ; j < fm.args.length ; j++ ) {
-               if ( fm.args[j].javaType.toString() == 'foam.core.X' ) {
-                  return false;
-               }
-               return true;
-             }
-             // return true;
-          }).map(function(m) { return m.name; }).sort();
-
-        if ( filteredMethod.length > 0 ) {
+        var methodName = methods.map(function(m) { return m.name; }).sort();
```

Comment:
I think a better name would be 'methodNames', since there are more than one name. The name 'methodName' is misleading since it suggests only one method name.

Reply:
done :) Thx

Reply:
@minsunny did you push the change to the name?